dev/io.openliberty.pages.4.0.internal.factories/bnd.bnd

```diff
@@ -0,0 +1,98 @@
+#*******************************************************************************
+# Copyright (c) 2023 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License 2.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-2.0/
+# 
+# SPDX-License-Identifier: EPL-2.0
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+javac.source: 17
+javac.target: 17
+
+Require-Capability: osgi.ee; filter:="(&(osgi.ee=JavaSE)(version=17))"
+
+Bundle-Name: Open Liberty Pages 4.0 Factories
+Bundle-SymbolicName: io.openliberty.pages.4.0.internal.factories
+Bundle-Description: WAS Pages 4.0, version ${bVersion}
+
+WLP-ServerName: IBM WebSphere Application Server
+WLP-ServerVersion: ${liberty.beta.version}
```

Comment:
Are these WLP- entries necessary? I only see them in OL in some of the Pages/Faces/WebContainer/Session bnd.bnd files. Perhaps we can get an issue opened to investigate / remove if possible?

Reply:
I'd be fine just opening an issue to see if we can clean this up.